src/Models/CatalogMember.js

```diff
@@ -48,7 +48,7 @@ var CatalogMember = function(application) {
      */
     this.isUserSupplied = true;
 
-    knockout.track(this, ['name', 'description', 'isUserSupplied']);
+    knockout.track(this, ['name', 'info', 'description', 'isUserSupplied']);
```

Comment:
`this.info` should be defined above as well.  Doing this _may_ improve the browser's ability to optimize the code (not that performance is a concern for this code), but more importantly it gives us a place to put the property's documentation.